index.html

```diff
@@ -0,0 +1,177 @@
+    </div>
+    </div>
+    <main id="programs">
+        <h2>Main Program</h2>
+        <div class="cards_list">
+            <div class="items">
+                <div class="cards">
+                    <div class="icon">
+                        <i class="fas fa-gifts"></i></div>
+                    <div class="image">Forum</div>
+                    <div class="text">Listen to the speakers from various countries about the messages of sharing and opening.</div>
+                </div>
+            </div>
+
+            <div class="items">
+                <div class="cards">
+                    <div class="icon"><i class="fab fa-wolf-pack-battalion"></i></div>
+                    <div class="image">Lecture</div>
+                    <div class="text">Listen to the speakers from various countries about the messages of sharing and opening.</div>
+                </div>
+            </div>
+
+            <div class="items">
+                <div class="cards">
+                    <div class="icon"><i class="fas fa-landmark"></i></div>
+                    <div class="image">Workshop</div>
+                    <div class="text">Listen to the speakers from various countries about the messages of sharing and opening.</div>
+                </div>
+            </div>
+
+            <div class="items">
+                <div class="cards">
+                    <div class="icon"><i class="fas fa-handshake"></i></div>
+                    <div class="image">Unity</div>
+                    <div class="text">Listen to the speakers from various countries about the messages of sharing and opening.</div>
+                </div>
+            </div>
+
+            <div class="items">
+                <div class="cards">
+                    <div class="icon"><i class="fas fa-people-carry"></i></div>
+                    <div class="image">
+                    
+                        Agriculture</div>
+                    <div class="text">Listen to the speakers from various countries about the messages of sharing and opening.</div>
+                </div>
+            </div>
+        </div>
+        <div class="main_button">Join Visit Rwanda</div>
+        <div class="main_button2"><a href="">SEE THE WHOLE PROGRAM</a> </div>
+    </main>
+
+    <div class="content" id="contents">
+        <h2 class="title_content">Featured speakers</h2>
+        <div class="hr"></div>
+        <div class="content_container">
+        <div class="content_item" id="item1">
+            <div class="image_content">
+                <img src="images/kagame.jpg" alt="">
+            </div>
+             <div class="text_content">
+                 <h2>Rwanda</h2>
+                 <h4>His President of the Republic of Rwanda. </h4>
+                 <h5>
+                    He served as Chair of the AU from 2018 to 2019 and chaired the East African Community from 2018-2021.
+                 </h5>
+             </div>
+            </div>
+        </div>
+    </div>
```

Comment:
- [ ] Typography (**font family**, **font-weight**, **font size**, **vertical height**, **font-style**...) and **spacing** of contents your _feature speakers_ section does not match the provided [design template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).  Kindly ensure to improve upon this aspects

**Your feature speakers section:**
![image](https://user-images.githubusercontent.com/12745474/133250445-169ea172-0bd9-46a2-b9ca-1a4523ed99f0.png)


**The original feature speakers section:**
![image](https://user-images.githubusercontent.com/12745474/132547059-14af2209-d5e7-48b5-9e41-622ed7e3ebea.png)